### PR TITLE
Add Seraph (neo4j for JS library)

### PR DESCRIPTION
**Pull request by SwamWithTurtles**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "help": "dt -h"
   },
   "dependencies": {
+    "dt": "0.0.1"
```
Comment (vvakame) [inline package.json]: what's this?

Comment (SwamWithTurtles) [inline package.json]: This is the 'diff tracker'. It wasn't included in the project so I had to install it -- I assumed that others would have the same problem down the line and I'm a big fan of reducing the effort involved in getting a project running the first time. Although I'm fine to rollback this commit if there's a reason that this isn't checked in

Comment (vvakame) [inline package.json]: ah, `dt` command is included in `definition-tester`.
Travis CI is passed in master/HEAD.
What version of npm do you use?

**Comment by mhegazy** [comment]
@SwamWithTurtles can you address the merge conflicts.

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**